src/helpers/editor.ts

```diff
@@ -991,11 +991,14 @@ export function notifyDragStarted(frameId?: number):void {
 
     // Add companion "image" (canvas) to the cursor - we use HTML2Canvas. 
     // The element to generate an image of is either the frame passed as argument
-    // or the shadow element containing the current selection.
-    const draggingEl = document.getElementById(getFrameUID(frameId??(useStore().frameObjects[useStore().selectedFrames[0]].parentId)));
-    if(draggingEl){
-        html2canvas(draggingEl, html2canvasOptions);
-    }
+    // or the selected frame's parent which will be cropped.
+    setTimeout(() => {
```

Comment:
Maybe just add a comment here about why we use the setTimeout (so that the grey rectangle shows up while html2canvas does its work, I believe?)